packages/service/src/service/base.ts

```diff
@@ -126,3 +138,11 @@ export class BtcAssetsApiBase implements BaseApis {
     this.token = token.token;
   }
 }
+
+function tryParseBody(body: unknown): Record<string, any> | undefined {
+  try {
+    return typeof body === 'string' ? JSON.parse(body) : void 0;
+  } catch {
+    return void 0;
```

Comment:
We should avoid using `void 0`, it is the same as undefined, but not as easy to understand.

Some libraries/packages use `void 0` because `undefined` can be shadowed by local variables, and "void 0" is shorter and can save a few bytes. 
But for modern JavaScript/TypeScript, this is a problem that does not require concern. Local variable shadowing cannot occur in TypeScript, and bundle size should be left to webpack/esbuild or other tools, rather than written as `void 0`

Reply:
I agree that undefined is better than void 0

Reply:
It's kinda a coding style habit for me, I use it because it's easy to type. 
I'll replace all `void 0` to `undefined` in another pull request.